inspirehep/modules/workflows/tasks/refextract.py

```diff
@@ -87,12 +91,24 @@ def extract_journal_info(obj, eng):
 
 
 @timeout(5 * 60)
-def extract_references(filepath, source=None, custom_kbs_file=None):
+def extract_references_from_PDF(filepath, source=None, custom_kbs_file=None):
     """Extract references from PDF and return in INSPIRE format."""
     extracted_references = extract_references_from_file(
         filepath,
         override_kbs_files=get_refextract_kbs_path(),
-        reference_format=u'{title},{volume},{page}'
+        reference_format=u'{title},{volume},{page}',
+    )
+
+    return map_refextract_to_schema(extracted_references, source=source)
+
+
+@timeout(5 * 60)
+def extract_references_from_form_field(text, source=None, custom_kbs_file=None):
```

Comment:
This function name is too specific: you only care about this being text, not that it comes from a form field.